setup.py

```diff
@@ -0,0 +1,54 @@
+from setuptools import setup, find_packages
+from distutils.util import convert_path
+
+
+def _read_long_description():
+    with open("README.md") as readme:
+        return readme.read()
+
+
+DEV_REQUIRE = [
+    "pytest",
+    "pytest-cov",
+    "black",
+    "mypy",
+    "python-semantic-release",
+]
+NAME = "flaky_tests_detection"
+NAME_DASHED = NAME.replace("_", "-")
+OLD_GIT_URL = "fliprate_actions"  # Are we going to change repo name?
+
+init_content = {}
+init_path = convert_path('flaky_tests_detection/__init__.py')
+with open(init_path) as init_file:
+    exec(init_file.read(), init_content)
+
+
+setup(
+    name=NAME_DASHED,
+    description="Github actions plugin to check flakiness of tests by calculating fliprates.",
+    long_description=_read_long_description(),
+    long_description_content_type="text/markdown",
+    author="Eero Kauhanen, Matvey Pashkovskiy, Alexey Vyskubov",
+    author_email="", # warning: check: missing meta-data: if 'author' supplied, 'author_email' must be supplied too
+    url=f"https://github.com/F-Secure/{OLD_GIT_URL}",  # Are we going to change repo name?
+    license="Apache License 2.0",  # Which license?
+    platforms="any",
+    version=init_content["__version__"],
+    packages=find_packages(exclude=[f"{NAME}.tests", f"{NAME}.tests.*"]),
+    entry_points={
+        "console_scripts": [
+            "check_flakes=flaky_tests_detection.check_flakes:main",  # What should be CLI name for entry point?
```

Comment:
@tatu-aalto, as the main user, what do you think?

Reply:
Had call with Tatu, _check_flakes_ -> _flaky_.